package.json

```diff
@@ -31,7 +31,9 @@
   },
   "homepage": "https://github.com/Rokt33r/remark-math#readme",
   "devDependencies": {
+    "babel-core": "^6.26.3",
```

Comment:
This might not actually be helpful, because it seems like you need babel-cli in the package directories anyway, and that requires babel-core.